components/measurement/CommonDetails.js

```diff
@@ -97,17 +100,42 @@ const CommonDetails = ({
     },
     {
       label: intl.formatMessage({ id: 'Measurement.CommonDetails.Label.Engine' }),
-      value: engine ?? intl.formatMessage({ id: 'Measurement.CommonDetails.Value.Unavailable' })
+      value: engine
```

Comment:
Should probably still display unavailable when it's not defined.

Reply:
This is a bit of cleanup, since the variable is already initialised as `unavailable` (just like `platform`) https://github.com/ooni/explorer/blob/eff50e47a35be23167671d133ae58facc8fbf270/components/measurement/CommonDetails.js#L69-L70

Reply:
Ah sorry I missed that. I think then this is good to merge!